### PR TITLE
ADEN-6155 Add missing F2 variables

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 1]
```diff
@@ -107,17 +107,20 @@ public static function onInstantGlobalsGetNewsAndStoriesVariables( array &$vars
 		$vars[] = 'wgAdDriverPorvataMoatTrackingCountries';
 		$vars[] = 'wgAdDriverPorvataMoatTrackingSampling';
 
-		// news&stories variables only
-		$vars[] = 'wgAdDriverInstartLogicF2RecoveryCountries';
+		// TODO: Remove after src=[gpt,ns] is finished
 		$vars[] = 'wgAdDriverNewsAndStoriesSingleSrcKeyValueCountries';
 		$vars[] = 'wgAdDriverNewsAndStoriesSrcKeyValueCountries';
+		// TODO: Remove after ADEN-6155
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php:119]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 2]
```diff
@@ -107,17 +107,20 @@ public static function onInstantGlobalsGetNewsAndStoriesVariables( array &$vars
 		$vars[] = 'wgAdDriverPorvataMoatTrackingCountries';
 		$vars[] = 'wgAdDriverPorvataMoatTrackingSampling';
 
-		// news&stories variables only
-		$vars[] = 'wgAdDriverInstartLogicF2RecoveryCountries';
+		// TODO: Remove after src=[gpt,ns] is finished
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 13 issues
* ![MAJOR][MAJOR] 10 major
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L136](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L136): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L137](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L137): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L211](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L211): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/4ea5d929e074ee2fc9712e02fcbe33a178792a07/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 13 issues
* ![MAJOR][MAJOR] 10 major
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L136](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L136): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L137](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L137): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L211](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L211): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/47f9f3fbb6948fc564a8ca51970ac2ba6facae43/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 13 issues
* ![MAJOR][MAJOR] 10 major
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L136](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L136): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L137](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L137): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L211](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L211): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/bbc468e251b8d523b63b742a76154ddf050569d7/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 3]
```diff
@@ -104,20 +104,23 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
...
 		$vars[] = 'wgAdDriverPorvataMoatTrackingCountries';
 		$vars[] = 'wgAdDriverPorvataMoatTrackingSampling';
+		$vars[] = 'wgAdDriverVideoMidrollCountries';
+		$vars[] = 'wgAdDriverVideoMoatTrackingCountries';
+		$vars[] = 'wgAdDriverVideoMoatTrackingSampling';
+		$vars[] = 'wgAdDriverVideoPostrollCountries';
 
-		// news&stories variables only
-		$vars[] = 'wgAdDriverInstartLogicF2RecoveryCountries';
+		// TODO: Remove after src=[gpt,ns] is finished
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php:116]: INFO Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 13 issues
* ![MAJOR][MAJOR] 10 major
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L18](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L18): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L136](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L136): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L137](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L137): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L211](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L211): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L235](https://github.com/Wikia/app/blob/93f159134d09f3c34e0fee68c7fb5fe63f976de3/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L235): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**